data/locale/en-US.ini

```diff
@@ -46,6 +46,8 @@ KeyFrames="Key Frames"
 KeyFrames.IntervalType="Interval Type"
 KeyFrames.IntervalType.Frames="Frames"
 KeyFrames.IntervalType.Seconds="Seconds"
+KeyFrames.IntervalType.Description="Keyframe interval type"
+KeyFrames.Interval.Description="Distance between key frames, in frames or seconds. Default is 2 seconds."
```

Comment:
Might be best to remove the default value here, as it may change and can be better shown by OBS's UI code instead of being put into the tooltip.

Reply:
OBS' UI doesn't offer an immediately apparent way to unset any changed settings to default, so I thought putting a note somewhere "if you broke it, here's how to unbreak" would help, but if you intend on putting it in later, this won't be useful.

Reply:
Yes, but there's a need for a UI rework in OBS anyway. Including default values shouldn't be an issue once the decisions have been made there.